public/app/features/alerting/unified/components/receivers/EditTemplateView.tsx

```diff
@@ -17,9 +17,9 @@ export const EditTemplateView: FC<Props> = ({ config, templateName, alertManager
 
   if (!template) {
     return (
-      <InfoBox severity="error" title="Template not found">
+      <Alert severity="error" title="Template not found">
         Sorry, this template does not seem to exit.
```

Comment:
typo (that we had before) :)

```suggestion
        Sorry, this template does not seem to exist.
```